samples/bluetooth/peripheral_hids_mouse/child_image/rpc_host.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_BT_L2CAP_TX_BUF_COUNT=5
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_DEVICE_NAME="Nordic_HIDS_mouse"
 CONFIG_BT_DEVICE_APPEARANCE=962
+CONFIG_BT_PRIVACY=y
```

Comment:
Can we set CONFIG_BT_PRIVACY in the Kconfig file that comes with the sample?

Reply:
Good point, done